tests/no_cycles.rb

```diff
@@ -50,4 +51,9 @@ def dfs(pkg)
   dfs(pkg)
 end
 
-puts "No dependency cycles found.".lightgreen
+if @failed == 0 || @failed > 1
+  @cycles = "cycles"
+else
+  @cycles = "cycle"
+end
```

Comment:
```
@cycles = "cycles"
@cycles = "cycle" if @failed == 1
```
2 lines  instead of 5.  1 conditional instead of 2.

Reply:
Good idea.

Reply:
What would make this even better is if you identified which dependencies of each package were causing the cycles.

Reply:
It does, it's just not obvious. This line is key: `    puts @path.to_s`

Reply:
Well it prints something but what if it displayed the package name along with the dependency for clarity?